src/test/modelinstance_test.cpp

```diff
@@ -360,6 +368,108 @@ TEST_F(TestLoadModel, CheckIfNonExistingBinFileReturnsFileInvalid) {
     EXPECT_EQ(status, ovms::StatusCode::FILE_INVALID) << status.string();
 }
 
+TEST_F(TestLoadModel, CheckSavedModelHandling) {
```

Comment:
Reuse test fixture:
TestWithTempDir() from test_utils.hpp

You need to inherit from that fixture, and then the temporary files are cleaned up after execution. You can check example usage in StressPipelineConfigChanges fixture.

Reply:
Additionally add test for loading such model and verifying at least the model state == AVAILABLE.